NEWS.rst

```diff
@@ -1,6 +1,11 @@
 New in Elpy 1.20.0
 ==================
 
+- Elpy now supports Black to format your code.
```

Comment:
There is no need to edit NEWS.rst, I do that usually by going through the commit messages when doing a release. That also reduces the confusion with the version numbers here. (Not that I mind people doing this … I'm open for ideas on how to format this? Maybe use "New in Elpy development" above?)

Reply:
"New in Elpy development" seems fine indeed.
I don't know how painful it is to add new features at each releases.
Keeping track of them as and when PR are merged may ease the process ?

I In fact, I realize that "adding en entry to NEW.rst" is one of the bullet point I added on the PR template (see the end of my first post in this PR).
We may want to get rid of that if you want to add entries at each release.

Reply:
We can have a separate PR that's only to update NEWS.rst as we go and merge once elpy has a new version. Then repeat the process again? Seems less burdensome that going through commit messages?